implementation/gate/src/test/java/eu/efti/eftigate/service/SearchWithConsignmentRequestDtoValidatorTest.java

```diff
@@ -15,7 +15,7 @@
 import static org.junit.jupiter.api.Assertions.assertFalse;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
-class SearchWithIdentifiersRequestDtoValidatorTest {
+class SearchWithConsignmentRequestDtoValidatorTest {
```

Comment:
same, i think it was not wanted 😄 
I would like to keep the clear separation between "Uil Request" and "Identifiers Request".
Since Consignement is common to both it will make things messy otherwise

Reply:
Reverted this in the next commit